Client/WarFare/UIVarious.cpp

```diff
@@ -652,6 +655,8 @@ bool CUIKnights::ReceiveMessage(CN3UIBase* pSender, uint32_t dwMsg)
 		NextPageButtonHandler();
 	else if (pSender->m_szID == "btn_clan_refresh")
 		RefreshButtonHandler();
+	else if (pSender->m_szID == "btn_Clan_party")
```

Comment:
These buttons are loaded. We don't need string comparisons here. We can just check the pointer, i.e.:
```cpp
else if (pSender == m_pBtn_ClanParty)
```